src/sentry/static/sentry/app/components/sidebar/discover2Item.tsx

```diff
@@ -67,24 +62,6 @@ class Discover2Item extends React.Component<Props, State> {
     browserHistory.push(target);
   };
 
-  handleEdit = (event: React.MouseEvent<Element>, item: SavedQuery) => {
-    event.preventDefault();
-    event.stopPropagation();
-    const {organization} = this.props;
-    const target = {
-      pathname: `/organizations/${organization.slug}/eventsv2/`,
-      query: {...EventView.fromSavedQuery(item).generateQueryStringObject(), edit: true},
-    };
-    browserHistory.push(target);
-  };
-
-  handleDelete = (event: React.MouseEvent<Element>, item: SavedQuery) => {
```

Comment:
Is this part of the sidebar deprecation plan? I like having the edit/delete closer to the name + save as they are contextually related.